docs/resources.html

```diff
@@ -214,25 +220,36 @@ <h2 id="improving-the-quality-of-reviews">Improving the quality of reviews</h2>
+				<p>Although hopefully very rare, there have been some reports of reviewer collusion and fraud. In some cases, authors set up fake accounts for suggested reviewers where they either <a href="https://www.chemistryworld.com/news/imposters-hijack-journals-peer-review-process-to-publish-substandard-papers/4013050.article" target="_blank">impersonate actual researchers</a> or create fictional academic characters in order to review their own papers (favorably, of course). Sometimes the <a href="https://retractionwatch.com/2015/07/08/30-papers-flagged-because-editors-may-have-subverted-the-peer-review-process-with-fake-accounts/" target="_blank">editor is even in on this scheme</a>. In other cases, authors form “<a href="https://cacm.acm.org/magazines/2021/6/252840-collusion-rings-threaten-the-integrity-of-computer-science-research/fulltext#FNA" target="_blank">collusion rings</a>” whereby they agree to suggest each other as reviewers and give favorable reviews to each other.</p>
 				<p>In order to avoid this, publishers can set up control systems that check for suspicious behavior such as private email addresses (although note that this is not foolproof as it will also flag innocent academics using non-institutional email addresses), as well as ensuring that papers always have at least one reviewer that has not been suggested by the authors themselves.</p>
+				<h2 id="submission-types">Submission types</h2>
+				<p>As well as traditional manuscript types (e.g., empirical articles, reviews and meta-analyses, and commentaries), several new innovative submission types are emerging. It may be  worth considering which submission types make sense for your journal, and whether it would be useful to add a new one!</p>
+				<p>Some journals offer Datasets / Data Descriptors as their own submission type (akin to the type of manuscripts published in <a href="https://www.nature.com/sdata/" target="_blank">Nature Scientific Data</a>). Journals offering this submission type will need to think through all the same decisions as they do when implementing a data and code availability policy for traditional articles (see our extensive section on this above), including ensuring data and code are hosted in a <a href="https://social-science-data-editors.github.io/guidance/Requested_information_hosting.html" target="_blank" title="List of trusted repositories">trusted repository</a>.</p>
```

Comment:
No need to do this now, but maybe consider italics for all journal titles in line with common style guides?

Reply:
good plan, will make it a point for the next round of edits!